docs/getting-started.md

```diff
@@ -266,22 +268,23 @@ account, for developing your own study:
     !!! note
         On Windows, installing Docker is usually
         straightforward, but can sometimes be complicated, depending on your exact
-        version and configure of Windows. If you run into problems, our more
+        version and configuration of Windows. If you run into problems, our more
         [detailed installation notes](install-docker.md) may help.
 
 
-    1. If you are using Windows 10 Pro, Enterprise or Education, you should [follow
-       these instructions](https://docs.docker.com/docker-for-windows/install/).
-       Otherwise, follow the instructions for [installing on Windows
-       Home](https://docs.docker.com/docker-for-windows/install-windows-home/).
+    1. Follow the instructions [here](https://docs.docker.com/docker-for-windows/install/) 
```

Comment:
The old link for installing on windows home just redirects to the general windows install page now

Reply:
We've a preference to use more decriptive links (ie, avoiding "here"). So this would be better as eg "follow [the Docker for Windows installation instructions](link...)."